### DISCUSSION TITLE
Synchronize React component state with local storage | Leo's dev blog – stories, insights, and ideas

### DISCUSSION THREAD

**Discussion by giscus** [hta218/leohuynh.dev#81] [Announcements]
# Synchronize React component state with local storage | Leo's dev blog – stories, insights, and ideas

Utilize a custom hook that syncs state with local storage, allowing for persistent state between sessions.

https://www.leohuynh.dev/snippets/sync-react-component-state-with-local-storage

**Comment by hta218** [comment]
#### Hey, thanks for checking out this post! 💡

If you found it useful (or have some ideas for improvement), I’d love to hear from you! Leave a comment below with your thoughts, or give a reaction to show your support. It means a lot to me!

Cheers 🍺
